features-json/fullscreen.json

```diff
@@ -306,7 +306,7 @@
       "15":"a x #2 #3",
       "15.1":"a x #2 #3",
       "15.2-15.3":"a x #2 #3",
-      "15.4":"a x #2 #3",
+      "15.4":"a x #3",
       "TP":"a x #2 #3"
```

Comment:
Can Note 2 regarding no `::backdrop` also be removed for TP? :)

Reply:
I would expect that the note should still be on the site because earlier versions of Safari still have that note applied. That's the pattern I've seen across Can I Use.

Reply:
Yes, note #2 should not apply to Safari Technology Preview.